tests/test_default_router.py

```diff
@@ -167,6 +178,32 @@ def test_variable(self):
         resource, method_map, params = self.router.find('/gists/42/raw')
         self.assertEqual(params, {'id': '42'})
 
+    def test_literal_vs_variable(self):
+        resource, method_map, params = self.router.find('/teams/default')
+        self.assertEqual(resource.resource_id, 19)
+
+        resource, method_map, params = self.router.find(
+            '/teams/default/members')
+        self.assertEqual(resource.resource_id, 7)
+
+        resource, method_map, params = self.router.find('/teams/foo')
+        self.assertEqual(resource.resource_id, 6)
+
+        resource, method_map, params = self.router.find('/teams/foo/members')
+        self.assertEqual(resource.resource_id, 7)
+
+        resource, method_map, params = self.router.find('/gists/first')
+        self.assertEqual(resource.resource_id, 20)
+
+        resource, method_map, params = self.router.find('/gists/first/raw')
+        self.assertEqual(resource.resource_id, 18)
+
+        resource, method_map, params = self.router.find('/gists/first/pdf')
+        self.assertEqual(resource.resource_id, 21)
+
+        resource, method_map, params = self.router.find('/gists/1776/pdf')
+        self.assertEqual(resource.resource_id, 21)
```

Comment:
Can we add test assertions for when a router *can't* find a route, instead of just for ones where there is a clear route defined?